### DISCUSSION TITLE
Official Node.js version support statement for Yo Teams

### DISCUSSION THREAD

**Discussion by andrewconnell** [pnp/generator-teams#313] [Q&A] [answered]
The docs state that you recommend the latest LTS version for the Yeoman generator, but don’t specify an exact version.

Should this be taken that you do support Node.js v16 which is the current ACTIVE LTS, the one listed on the Node.js homepage?

**Comment by wictorwilen** [comment, marked as answer]
Good question. We're testing on 12, 14, 15 and 16 as of now. So yes, 16 should be "supported" (as in supported on a non-supported project...you know what I mean)

**Reply by wictorwilen** [reply]
And...just to add, for upcoming releases I'm removing test on Node 12...

**Reply by andrewconnell** [reply]
Thanks for the clarification. So... back to my original question, when your guidance is Node LTS, you mean [active & maint versions](https://nodejs.org/en/about/releases/), just like the Node guidance is "production should use active & maint", correct?

**Reply by wictorwilen** [reply]
Yes, that's the intention - Active or Maintenance versions
